grc20/app/src/services/erc20/mod.rs

```diff
@@ -0,0 +1,166 @@
+use core::fmt::Debug;
+use gstd::{collections::HashMap, format, msg, ActorId, Decode, Encode, String, TypeInfo, Vec};
+use primitive_types::U256;
+use sails_rtl::gstd::gservice;
+
+pub use utils::*;
+
+pub mod funcs;
+pub(crate) mod utils;
+
+static mut PROGRAM: Option<Program> = None;
+
+#[derive(Debug, Default)]
+pub struct Program {
```

Comment:
This seems quite confusing. You already have a concept of program in your contract which is supposed to host your services, then one encounters yet another concept with the same name, but it is used for holding data services work on.